src/adapter.ts

```diff
@@ -230,6 +234,19 @@ export default class JestTestAdapter implements TestAdapter {
     }
   }
 
+  private getOrCreateOutputWindowForProject(project: ProjectRootNode): vscode.OutputChannel {
+    const channelName = `Jest (${project.config.projectName})`;
+    const channel = this.outputChannels.find(c => c.name === channelName);
+
+    if (channel) {
+      return channel;
+    }
+
+    const created = vscode.window.createOutputChannel(channelName);
```

Comment:
I gave this a whirl, and it took me a moment to find where the output goes.  This is worth a README entry and perhaps something in the extension page (if that differs from the README - I forget how that page is generated).

Reply:
Cool, I have added a small note to the Readme (which updates the extension page too).  The documentation is something that is a little lacking, but I wanted to get the extension more mature before fully documenting it.

Reply:
Still not seeing this change in the PR either.